docker/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM debian:jessie
+
+ENV PYTHONIOENCODING UTF-8
+
+# Pypy is installed from a package manager because it takes so long to build.
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    curl \
+    git \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    libmemcached-dev \
+    libncurses5-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    pkg-config \
+    pypy \
+    wget \
+    zlib1g-dev
+
+# Install couchbase
+RUN wget http://packages.couchbase.com/clients/c/libcouchbase-2.8.4_jessie_amd64.tar \
+    && tar -vxf libcouchbase-2.8.4_jessie_amd64.tar \
+    && dpkg -i libcouchbase-2.8.4_jessie_amd64/libcouchbase2-core_2.8.4-1_amd64.deb \
+    && dpkg -i libcouchbase-2.8.4_jessie_amd64/libcouchbase-dev_2.8.4-1_amd64.deb
+
+# Setup our home environment
+ENV CELERY_USER someone
+RUN addgroup --gid 1000 ${CELERY_USER}
+RUN adduser --system --disabled-password --uid 1000 --gid 1000 ${CELERY_USER}
+RUN usermod -aG sudo ${CELERY_USER}
```

Comment:
Is it necessary to promote the user to sudoer?

Reply:
I'll remove this. I did this for debugging.